spec/rubocop/cop/rails/strong_parameters_expect_spec.rb

```diff
@@ -90,12 +104,12 @@
         )
       RUBY
 
-      expect_correction(<<~RUBY)
-        params.expect(
-          user: [:name, # comment
-          :age] # comment
-        )
-      RUBY
+      # expect_correction(<<~RUBY)
```

Comment:
I unfortunately did not find a solution to preserve comments :/  I might need help here